test/Microsoft.OpenApi.Tests/Validations/OpenApiResponseValidationTests.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. 
+
+using System.Collections.Generic;
+using Microsoft.OpenApi.Models;
+using Xunit;
+
+namespace Microsoft.OpenApi.Validations.Tests
+{
+    public class OpenApiResponseValidationTests
+    {
+        [Fact]
+        public void ValidateDescriptionIsRequiredInResponse()
+        {
+            // Arrange
+            IEnumerable<ValidationError> errors;
+            OpenApiResponse response = new OpenApiResponse();
+
+            // Act
+            bool result = response.Validate(out errors);
+
+            // Assert
+            Assert.False(result);
+            Assert.NotNull(errors);
+            ValidationError error = Assert.Single(errors);
+            Assert.Equal("The field 'description' in 'response' object is REQUIRED.", error.ErrorMessage);
```

Comment:
Should we check that the path and the reason are also as expected?

Reply:
Good catch. I added such here.

Such test cases is helping us to understand the design. I'd like to make sure we can finalize the design first.

---
In reply to: [157651079](https://github.com/Microsoft/OpenAPI.NET/pull/152#discussion_r157651079) [](ancestors = 157651079)